packages/orchestration/src/fetched-chain-info.js

```diff
@@ -207,42 +228,42 @@ export default /** @type {const} } */ ({
           version: 'ics20-1',
         },
       },
-      'stargaze-1': {
-        id: 'connection-56',
-        client_id: '07-tendermint-86',
+      'stride-1': {
```

Comment:
the change in order here is kinda... distracting

Reply:
good point. fixed